scripts/rssFeedGen.py

```diff
@@ -0,0 +1,95 @@
+from bs4 import BeautifulSoup
+from datetime import datetime
+import time
+from email.Utils import formatdate
+import PyRSS2Gen
+import StringIO
+import re
+
+def extract_releases(html):
+    html_doc = open(html, 'r').read()
```

Comment:
Usually better to do:
```python
with open(html, 'r') as f:
    soup = BeautifulSoup(f.read(), "lxml")
```

This way the file is closed automatically when you are out of the `with` context.